src/test/java/gradle/cucumber/ServerProcess.java

```diff
@@ -0,0 +1,18 @@
+package gradle.cucumber;
+
+import java.io.IOException;
+
+public class ServerProcess {
+    Process serverProcess;
+
+    public static ServerProcess start() throws IOException{
+        return new ServerProcess();
+    }
+
+    private ServerProcess() throws IOException {
+        serverProcess = new ProcessBuilder()
+                .command("java", "-jar", "build/libs/com.olgaboiar.guacomole-1.0-SNAPSHOT.jar")
```

Comment:
Can we make the location of the executable configurable?

Reply:
implemented in https://github.com/olgaboiar/GuacamoleServer/pull/2/commits/4155dd3e702a428653f375a6b052f1b246ef684e